reviews_ratings.py

```diff
@@ -0,0 +1,110 @@
+from flask import Flask, request, jsonify
+import sqlite3
+import time
+
+app = Flask(__name__)
+
+def get_db():
+    conn = sqlite3.connect('reviews.db')
+    return conn
+
+def init_db():
+    conn = get_db()
+    cursor = conn.cursor()
+    cursor.execute('''
+        CREATE TABLE IF NOT EXISTS reviews (
+            review_id INTEGER PRIMARY KEY,
+            product_id TEXT,
+            user_id TEXT,
+            review TEXT
+        )
+    ''')
+    cursor.execute('''
+        CREATE TABLE IF NOT EXISTS ratings (
+            rating_id INTEGER PRIMARY KEY,
+            product_id TEXT,
+            user_id TEXT,
+            rating INTEGER
+        )
+    ''')
+    conn.commit()
+    conn.close()
+
+init_db()
+
+@app.route('/add_review', methods=['POST'])
+def add_review():
+    data = request.json
+    conn = get_db()
+    cursor = conn.cursor()
+    query = f"""
+        INSERT INTO reviews (product_id, user_id, review) 
+        VALUES ('{data['product_id']}', '{data['user_id']}', '{data['review']}')
```

Comment:
**Security Issue**: The code is vulnerable to SQL injection due to the direct interpolation of user input into SQL queries. This can allow an attacker to execute arbitrary SQL code. <br> **Fix**: Use parameterized queries to prevent SQL injection. <br> **Code Suggestion**: 
 ```
 -        VALUES ('{data['product_id']}', '{data['user_id']}', '{data['review']}')
 +        VALUES (?, ?, ?)
 ```